src/server/routes/backend-wallet/sendTransaction.ts

```diff
@@ -71,19 +71,37 @@ export async function sendTransaction(fastify: FastifyInstance) {
       const {
         "x-backend-wallet-address": fromAddress,
         "x-idempotency-key": idempotencyKey,
-      } = request.headers as Static<typeof walletHeaderSchema>;
+        "x-account-address": accountAddress,
+      } = request.headers as Static<typeof walletWithAAHeaderSchema>;
       const chainId = await getChainIdFromChain(chain);
 
-      const { id: queueId } = await queueTxRaw({
-        chainId: chainId.toString(),
-        fromAddress,
-        toAddress,
-        data,
-        value,
-        simulateTx,
-        idempotencyKey,
-        ...txOverrides,
-      });
+      let queueId: string;
+      if (accountAddress) {
+        const { id } = await queueTxRaw({
+          chainId: chainId.toString(),
+          signerAddress: fromAddress,
+          accountAddress,
+          target: toAddress,
```

Comment:
arent these all optional? prob dont need the if / else right? just pass everything through even if undefined

Reply:
There's some code like this:
```typescript
if (tx.accountAddress && tx.signerAddress) {
  // process as a userOp
}
```

In this case accountAddress is undefined so it would be okay, but I'm concerned about any stray code that tries to handle a userOp if signerAddress is set (and in this case we'd be setting signerAddress for EOA transactions too).